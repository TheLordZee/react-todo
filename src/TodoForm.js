
const TodoForm = ({handleSubmit, onChange, formData}) => (
    <form className="TodoForm" onSubmit={handleSubmit}>
        <label htmlFor="text" hidden>Text</label>
        <input 
            name="text"
            className="TodoForm-input"
            onChange={onChange}
            value={formData.todoText}
        />
        <button>Add Todo</button>
    </form>
)

export default TodoForm;