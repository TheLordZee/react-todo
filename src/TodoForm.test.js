import { render } from '@testing-library/react';
import TodoForm from "./TodoForm"
const data = {
    todoTex: "Test todo"
}

it('renders', () => {
  render(<TodoForm  formData={data}/>);
});

it("matches snapshot", () => {
    const {asFragment} = render(<TodoForm formData={data} />);
    expect(asFragment()).toMatchSnapshot();
})