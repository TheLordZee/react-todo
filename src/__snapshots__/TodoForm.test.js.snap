// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <form
    class="TodoForm"
  >
    <label
      for="text"
      hidden=""
    >
      Text
    </label>
    <input
      class="TodoForm-input"
      name="text"
      value=""
    />
    <button>
      Add Todo
    </button>
  </form>
</DocumentFragment>
`;
